docs/src/content/docs/ru/challenges/angular/8-pipe-pure.md

```diff
@@ -0,0 +1,22 @@
+---
+title: 🟢 Чистые пайпы
+description: Испытание 8 про создание чистого пайпа
+author: thomas-laforge
+challengeNumber: 8
+command: angular-pipe-easy
+blogLink: https://medium.com/ngconf/deep-dive-into-angular-pipes-c040588cd15d
+sidebar:
+  order: 3
+---
+
+Цель этой серии из 3 задач на тему пайпов - освоить работу с **пайпами** в Angular.
```

Comment:
The docs for this challenge was updated after your pull request, if possible please check out this challenge again and update the translation.

Reply:
Sure. Did it here https://github.com/tomalaforge/angular-challenges/pull/609